### ISSUE TITLE
scikit-learn 및 FastAPI 기반 모델 배포 기능 구현

### ISSUE THREAD

**Issue by tyjeon24** [f-lab-edu/anomaly-factory#3]
# 목표
* 로컬 환경에서 접근할 수 있는 scikit-learn API 모델을 구축한다.
* 사용자는 127.0.0.1:포트로 서비스에 접근하여, 신규 데이터에 대한 분류 결과를 확인할 수 있다.

# 상세
* 데이터 : sklearn.datasets.load_iris의 load_iris를 사용한다.
  * Naver Cloud 신청 완료 전까지는 load_iris로 FastAPI의 작동 상태 확인 후, 대용량 데이터셋에 대해 적용 예정
  * 후보 데이터 1 : [Large-scale Energy Anomaly Detection (LEAD)](https://www.kaggle.com/competitions/energy-anomaly-detection)
    * 데이터 1GB 이상이며 레이블 있음(분류 모델 metric 적용 가능)
* 모델 : scikit-learn에서 지원하는 RandomForestClassifier 모델을 사용한다.
  * 다른 모델의 추가는 BentoML로 API 배포 방법 변경 후 작업 예정
